CHANGELOG.md

```diff
@@ -0,0 +1,50 @@
+# Change Log
```

Comment:
I'm a little confused by this.  I think this document should be the actual change log, not just guidelines for how to write a change log?    So for this PR, I'd say we should have a changelog that corresponds to v1.0.0 .  Also I think we should explore using auto-generators for changelogs.  I never have, but let's try it out!  There are some good resources here: https://www.freecodecamp.org/news/a-beginners-guide-to-git-what-is-a-changelog-and-how-to-generate-it/

Reply:
If we want to add guidelines for how to write/generate the change log, let's add them to the release guide - you've already got most of it.

Reply:
@kheal there this outlines how the change log is managed and how links to how it is stylized based on standards that already exist. I advocate for leaving this here since it is very much part of the release process. I can change the heading to say `Change log guide` or something if that makes it clearer.

I did not see the point in auto-generating a changelog since the release will autogenerate the list of PRs that were merged. Many of those auto-generating changelogs will just create a list of all the commits which is uninformative to most readers, messy (especially if you are like me and commit every change you make). I was thinking a nice clean hand-curated change log is the way to go. Maybe we can discuss at our next meeting.

Reply:
Sorry, that highlight portion is the actual change log. We don't have any changes yet since we don't have a release. I added it so we could all see how it is structured.

Reply:
I can create a release as part of this PR so it makes sense.

Reply:
This all makes sense.  I think if we change the name or location of the guidelines (I agree they are worthwhile to keep in the repo) and add the first entry for the CHANGELOG that'll be great.

Reply:
Let's add a section to the CONTRIBUTING.md guide and say that it's for managers only.

Reply:
It says that on line 4